en/08_Changelogs/6.0.0.md

```diff
@@ -891,6 +913,51 @@ As part of these changes [`ArrayList::find()`](api:SilverStripe\Model\List\Array
 
 ## Other changes
 
+### Campaign admin removed from the CMS {#campaign-admin-removed}
+
+The `silverstripe/campaign-admin` was a core module that provided a way to publish multiple related records at the same time, but it was very rarely used in practice. It has been removed from the default CMS installation and has had its commercial support removed. Its integration support has been removed from `silverstripe/cms`, `silverstripe/admin`, and `silverstripe/asset-admin`, all of which provided "Add to campaign" UX functionality. This means that it will not be possible to get any sort of campaign admin functionality in CMS 6 even if using a forked version of `silverstripe/campaign-admin`.
+
+### Modules loosing commercial support {#modules-loosing-support}
```

Comment:
Move this to be at the top, to be consistent with the [CMS 5 changelog](https://docs.silverstripe.org/en/5/changelogs/5.0.0/).

Give it the same headings as we used in CMS (with the obvious exclusion of "Modules with upgrade guides" since we haven't done any). Move the "Campaign admin removed from the CMS" section to be under "Change to supported modules"

Reply:
Done